.husky/commit-msg

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# This script checks if the commit message follows a consistent format(usually useful for consistency and devops)
+if ! head -1 "$1" | grep -qE "^(feat|fix|ci|chore|docs|test|style|refactor|perf|build|revert)(\(.+?\))?: .{1,}$"; then
```

Comment:
We should allow `!` after the keyword so we can mark features/chores/etc. as breaking changes without forcing long-form messages/a second line in the commit content

Reply:
You mean something like 

`feat!: migrate wallet to use selectors`

Where `!` indicates that it's a breaking change?

Reply:
Yup, per https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with--to-draw-attention-to-breaking-change :)